azure-pipelines-1es.yml

```
@@ -0,0 +1,126 @@
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+
+# Branches that trigger a build on commit
+trigger:
+- main
+- release/*
+- internal/release/*
+variables:
+- group: AzureDevOps-Artifact-Feeds-Pats
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    sdl:
+      sourceAnalysisPool: NetCore1ESPool-Svc-Internal
+    pool:
+      name: AzurePipelines-EO
+      image: AzurePipelinesWindows2022compliantGPT
```

I think we should use something like this:
```suggestion
      name: netcore1espool-internal
      image: windows.vs2022preview.amd64
```